applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -105,6 +105,9 @@ def __build_cmd_get(core: Core, device: AudioDevice, build: BuildType, pristine,
         if options.min_b0n:
             device_flag += " -DCONFIG_B0N_MINIMAL=y"
 
+        if options.shield:
```

Comment:
This is not a general shield, so the option can be more specific.

Reply:
Will change to nRF21540